decidim-elections/app/constraints/decidim/votings/current_voting.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    # This class infers the current voting we're scoped to by
+    # looking at the request parameters and the organization in the request
+    # environment, and injects it into the environment.
+    class CurrentVoting
+      # Public: Matches the request against an voting and injects it
+      #         into the environment.
+      #
+      # request - The request that holds the voting relevant
+      #           information.
+      #
+      # Returns a true if the request matched, false otherwise
+      def matches?(request)
+        env = request.env
+
+        @organization = env["decidim.current_organization"]
+        return false unless @organization
+
+        current_voting(env, request.params) ? true : false
```

Comment:
If the `current_voting` is just returning some kind of object you can do something like:
```
current_voting(env, request.params).present?
```

I think it looks a bit better... the ternary operator with just `true` and `false` looks weird to me 😄